### PR TITLE
Fix DistribtuedLoad CLI Output and dump failed files to local csv

### PR DESCRIPTION
**Pull request by jja725**
### What changes are proposed in this pull request?

Fix the CLI output of distributedLoad command 
### Why are the changes needed?

With batch, the output of distributedLoad command output is having wrong file count. And there's no way for user to parse the output and get failures. S

### Does this PR introduce any user facing changes?

  1. change in user-facing CLI
If there's failure in the execution, there would be additional output as the following:
Here are recent failed files:
/a/b/c,
/a/b/d,
Check out logs/user/distributedLoad_`PATH`_failures.csv for full list of failed files 
(PATH here would be the underscore separated path of distributedLoad command, for example if path is /a/b/c, then the failure file would be in logs/user/distributedLoad_a_b_c_failures.csv)
Output file format would be one failure file name per line without header:
/a/b/c
/c/b/a

### CODE REVIEW

**Comment by yuzhu** [comment]
@jja725 could you include a sample output in the description of the PR?

**Comment by jja725** [comment]
the test is action timeout due to installing tarball, which seems irrelevant. added output in the description.

**Comment by jja725** [comment]
Error: 8.301 [ERROR] Tests run: 8, Failures: 2, Errors: 0, Skipped: 0, Time elapsed: 3.877 s <<< FAILURE! - in alluxio.util.JvmPauseMonitorTest
Error: 8.307 [ERROR] alluxio.util.JvmPauseMonitorTest.testMockedInfoPause  Time elapsed: 1.506 s  <<< FAILURE!
java.lang.AssertionError

**Comment by jja725** [comment]
Error: 1.529 [ERROR] Errors: 
Error: 1.530 [ERROR] alluxio.client.cli.JournalToolTest.dumpHeapCheckpointFromEmbeddedJournal
Error: 1.530 [ERROR]   Run 1: JournalToolTest.dumpHeapCheckpointFromEmbeddedJournal » Runtime java.lang.Runt...
Error: 1.530 [ERROR]   Run 2: JournalToolTest.dumpHeapCheckpointFromEmbeddedJournal » Runtime java.lang.Runt...
21:08:21.531 [INFO] 
Warning: 531 [WARNING] Flakes: 
Warning: 531 [WARNING] alluxio.client.cli.job.DistributedCommandsStatsTest.testAsyncPersistCancelStats
Error: 1.531 [ERROR]   Run 1: DistributedCommandsStatsTest.testAsyncPersistCancelStats:197 » Timeout

File: tests/src/test/java/alluxio/client/cli/fs/command/DistributedLoadCommandTest.java [hunk 1]
```diff
@@ -250,4 +249,45 @@ public void loadDirWithPassiveCacheInBatch() throws IOException, AlluxioExceptio
...
+        boolean result = new File(path).delete();
+        Assert.assertTrue(result);
+      }
+    }
+    fsShell.run("distributedLoad", "/testFailure");
+    Assert.assertTrue(mOutput.toString().contains(
+        String.format("Completed count is %s,Failed count is %s.\n", fileSize / 2, fileSize / 2)));
+    Assert.assertTrue(mOutput.toString().contains(
+        "./logs/user/distributedLoad_testFailure_failures.csv for full list of failed files."));
+  }
```
Comment (yuzhu) [inline tests/src/test/java/alluxio/client/cli/fs/command/DistributedLoadCommandTest.java:292]: can we read the csv and make sure the expected failures are in there?

Comment (jja725) [reply tests/src/test/java/alluxio/client/cli/fs/command/DistributedLoadCommandTest.java:300]: added

**Review by yuzhu — APPROVED** [review]

**Comment by yuzhu** [comment]
alluxio-bot, merge this please

**Comment by jiacheliu3** [comment]
alluxio-bot, cherry-pick this to branch-2.7 please

**Pull request merged**